README.md

```diff
@@ -3,20 +3,20 @@
 
 ## Running examples
 
-To run an example customisation you need an exiting OpenShift cluster with integreatly installed.
+To run an example customisation you need an existing OpenShift cluster with integreatly installed.
 
 Next you need to get the available inventory file from the manifest secret
 
 ```
 (mac osx)
 
-oc get secret inventory -n webapp --template '{{index .data "generated_inventory"}}'  | base64 -D > inventory
+oc get secret manifest -n webapp --template '{{index .data "generated_inventory"}}'  | base64 -D > inventory
```

Comment:
oc get secret inventory -n webapp --template '{{index .data "generated_inventory"}}'  | base64 -d > inventory

gives the following error

Error from server (NotFound): secrets "inventory" not found


Guessing solution above

Reply:
@maleck13 is the secret going to be 'manifest' or 'inventory'

Reply:
@finp Is this from master. There are two secrets. The manifest and the inventory